docs/src/index.md

```diff
@@ -62,16 +67,18 @@ julia> using PackageAnalyzer
 
 julia> analyze(PackageAnalyzer)
 Package PackageAnalyzer:
-  * repo:
+  * repo: 
```

Comment:
Why there is this extra space (here and other snippets elsewhere)? :thinking:

Reply:
That's how our `show` method works: `* repo: $(p.repo)`. If `p.repo == ""` then we get that space still. Not sure why it wasn't here before, I didn't touch that bit, just copy-pasted the new ones. Maybe we manually removed it before?